### ISSUE TITLE
Some data structure update.

### ISSUE THREAD

**Issue by Pt-Djefferson** [FINDarkside/TLD-Save-Editor#18]
Dunno how it was, but now need to change some data order
GlobalSaveGameFormat.cs:

```
        [Deserialize("m_GameManagerSerialized", true)]
-        public SceneTransitionData SceneTransition { get; set; }
+        public GameManagerData GameManager { get; set; }
        [Deserialize("m_HudManagerSerialized", true)]

```
GlobalData.cs:

```
        public Dictionary<string, byte[]> m_Dict { get; set; }
    }

+    public class GameManagerData
+    {
+        public bool m_BlockAbilityToRest { get; set; }
+        public string m_BlockedRestLocID { get; set; }
+        [Deserialize("m_SceneTransitionDataSerialized", true)]
+        public SceneTransitionData SceneTransition { get; set; }
+    }
+
    public class SceneTransitionData

```

**Comment by Pt-Djefferson** [comment]
```
    public class AuroraScreenInfo
    {
        public string m_PrefabName { get; set; }
    }

```
in that section m_PrefabName  no longer used in real saves.

**Comment by FINDarkside** [comment]
Good catch, not sure when this changed as the game still accepts the old format. Fixed with 06f513ee93260cf655da462f333d93c42649b01a and 1b3fa912aabb7f659f07ea7cf65358221ecd50b1. Removed the aurora structure completely as it's not used for anything in the editor at the moment.
